openapi/src/json-schemas/_questionnaires_{questionnaireId}_submissions/post/req/201.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["data"],
+    "properties": {
+        "data": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["type", "attributes"],
+            "properties": {
+                "type": {
+                    "const": "submissions"
+                },
+                "attributes": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "questionnaireId": {
+                            "type": "string",
```

Comment:
@sinclairs 

At the moment no information in the request body of this POST request (`/questionnaires/{questionnaireId}/submissions`) is being used at the DCS side. The request body is completely ignored as it is not needed to know how to proceed.

Having said this, it is best practice (although not required) to have a request body in a POST request, so I want to include one to conform to best practice, but I don't need one, so what's the point in having one?

Should I change it to make the request body a useful entity? or change the request VERB (not sure what to, mind you)? or go against best practice?